src/wren_core.c

```diff
@@ -888,6 +906,26 @@ DEF_PRIMITIVE(num_isNan)
   RETURN_BOOL(isnan(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_rad)
+{
+  RETURN_NUM(floor(AS_NUM(args[0]) / 57.2957795130823208768));
+}
+
+DEF_PRIMITIVE(num_sign)
+{
+  double value = AS_NUM(args[0]);
+  double sign;
```

Comment:
Instead of using a temp variable, I'd probably just return directly from within each branch.